packages/material-ui/src/AppBar/AppBar.js

```diff
@@ -41,17 +42,20 @@ export const styles = (theme) => {
     positionSticky: {
       // ⚠️ sticky is not supported by IE11.
       position: 'sticky',
+      zIndex: theme.zIndex.appBar,
       top: 0,
       left: 'auto',
       right: 0,
     },
     /* Styles applied to the root element if `position="static"`. */
     positionStatic: {
       position: 'static',
+      zIndex: 'inherit',
```

Comment:
Rather than removing `zIndex`, we could give `zIndex: inherit`?


https://codepen.io/sujinlee/pen/QWEmBGR